plugins/node.d/mbmon_.in

```diff
@@ -16,14 +16,24 @@
 # implied warranties, including, without limitation, the implied
 # warranties of merchantability and fitness for a particular
 # purpose.
+
+# Needs following minimal configuration in plugin-conf.d/munin-node:
+#   [mbmon_*]
+#   env.mbmonpath /usr/local/bin/mbmon
+#   env.mbmonargs -I -pit87
+#
+# Parameters
+#       mbmonpath     - Specify path to smartctl program
+#       mbmonargs     - Added argument passed to mbmon with '-c 1 -r '+mbmonargs
+#
+# If it is not automatically recognized, the individual parameters are appended.
 #
 
 #%# family=contrib
 #%# capabilities=autoconf suggest
 
 what=$(basename "$0" | sed 's/^mbmon_//g')
-mbmon=${mbmon:-$(which mbmon)}
-
+mbmon="${mbmonpath:-$(which mbmon)} -c1 -r ${mbmonargs}"
```

Comment:
Could you please elaborate a bit, how this changes the behaviour of the plugin for its current users?

Did everyone need to specify the `mbmon` variable as `/usr/local/bin/mbmon -c 1 -r`? Or did the previous version also work with `mbmon` being empty?
(I do not use this plugin, thus I have no idea, how people are using it)

How about keeping the previous `mbmon` definition and simply replace `-c 1 -r` in every `mbmon` call below with your new configurable `${mbmonargs}` variable?

Reply:
At least, in an environment where a plurality of sensors are attached, there is a case where the correct value is not detected.
In the case of one sensor, it works without problem scan to specify that it is operating even in the sky.

It's smarter to append '-c 1 -r' to the default of ${mbmonargs}.
I agree with the replacement.